tardis/plasma/properties/atomic.py

```diff
@@ -153,6 +153,8 @@ class ZetaData(BaseAtomicDataProperty):
     outputs = ('zeta_data',)
 
     def _filter_atomic_property(self, zeta_data, selected_atoms):
+        for temperature in range(42000, 82000, 2000):
```

Comment:
We should extrapolate and not increase the storage. I mean in the end how far it is outside -> outside means use the last valid value